index.html

```diff
@@ -23,10 +23,11 @@
 </script>
 </head>
 <body>
```

Comment:
The whole web page is blur so I was unable to check the project against the requirements
![blur](https://user-images.githubusercontent.com/35504149/137470527-04571208-4339-43c8-a4dd-dbc2d8e312b6.png)
.